### PR TITLE
User Accounts and Logger

### PR DESCRIPTION
**Pull request by danielstanojkov-quantox**

### CODE REVIEW

File: app/Http/Controllers/Api.php [hunk 1]
```diff
@@ -22,6 +23,7 @@ public function results()
         } catch (\Throwable $th) {
             header('HTTP/1.1 500 Internal Server Error');
             header('Content-Type: application/json; charset=UTF-8');
+            Log::error($th->getMessage());
             die(json_encode(array('message' => $th->getMessage(), 'code' => 500)));
```
Comment (davorminchorov) [inline app/Http/Controllers/Api.php]: You can use [] instead of array.

File: app/Http/Controllers/Dashboard.php [hunk 2]
```diff
@@ -50,12 +52,31 @@ protected function getSidebarData(): array
       'host' => Auth::host(),
       'username' => Auth::username(),
       'databases' => $db->allDatabaseNames(),
-      'tables' => $tables
+      'tables' => $tables,
+      'accounts' => $this->getUsers()
     ];
 
+    // var_dump($data['accounts']);
+    // die;
```
Comment (davorminchorov) [inline app/Http/Controllers/Dashboard.php]: you can remove these lines.

File: app/Http/Requests/CreateUserRequest.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+     * @return bool 
+     */
+    public static function validate(mixed $role, string $username): bool
+    {
+        if (empty(trim($username))) {
+            Session::flash('registration_failed', 'Please enter your username.');
+            return false;
+        }
+
+        if (is_null($role)) {
```
Comment (davorminchorov) [inline app/Http/Requests/CreateUserRequest.php]: `is_null($role) -> !$role`

**Review by davorminchorov — APPROVED** [review]
Looks good.

**Review by internalle — APPROVED** [review]

**Pull request merged**